setup.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# Verify that poetry is installed correctly.
+poetry --version >/dev/null 2>&1
+if [ $? -ne 0 ]; then
+    echo 'Poetry may not be installed correctly. Verify that poetry is in your $PATH.'
```

Comment:
Perhaps there could be some extra troubleshooting output like:
Check your .zshrc or .bashrc contains export PATH="$HOME/.local/bin:$PATH"
If it is there try `source ~/.zshrc` or `source ~/.bashrc`
Check if you have python3 installed by running `python3 --version` and try installing poetry again using:
curl -sSL https://install.python-poetry.org | python3 -

Merely a suggestion :)